packages/@lightningjs/ui-components/src/components/Control/ControlSmall.styles.js

```diff
@@ -34,7 +34,6 @@ export const base = theme => {
     },
     prefixH: theme.spacer.md * 5,
     radius,
-    textStyle: theme.typography.button2,
```

Comment:
Deleted this because it is already defined in ButtonSmall component which it extends.

Reply:
we'd still need it because otherwise it's not included in the returned object right?